docs/content/policy-reference.md

```diff
@@ -324,6 +324,7 @@ complex types.
 | Built-in | Description | Wasm Support |
 | ------- |-------------|---------------|
 | <span class="opa-keep-it-together">``output := array.concat(array, array)``</span> | ``output`` is the result of concatenating the two input arrays together. | ✅ |
+| <span class="opa-keep-it-together">``output := array.reverse(array)``</span> | ``output`` is the result of reversing the order of the elements in ``array``. | ✅ |
```

Comment:
This doesn't have Wasm support yet, so should be "SDK-dependent"

Reply:
I will see if I can make this claim true, but I will of course change this if it is does not get included in this PR.

Reply:
Based on @srenatus pointer above, I've tried adding implementations of WASM for both functions with some tests. Let me know if you see some problems or more tests are needed.

Reply:
Awesome! @srenatus will be better equipped to review that part :)